packages/yoastseo/src/languageProcessing/helpers/word/getWords.js

```diff
@@ -3,33 +3,36 @@ import sanitizeString from "../sanitize/sanitizeString";
 import { filter, flatMap } from "lodash-es";
 import removePunctuation, { punctuationRegexString } from "../sanitize/removePunctuation.js";
 
-const interJectionRegexString = `([${punctuationRegexString}])`;
-const interJectionRegex = new RegExp( interJectionRegexString, "g" );
+const punctuationRegexStr = `([${punctuationRegexString}])`;
+const punctuationRegex = new RegExp( punctuationRegexStr, "g" );
 
 /**
  * Returns an array with words used in the text.
  *
  * @param {string} text The text to be counted.
+ * @param {string} [wordBoundaryRegexString=\\s] The regex string for the word boundary that should be used to split the text into words.
  * @param {boolean} [shouldRemovePunctuation=true] If punctuation should be removed. Defaults to `true`.
  *
  * @returns {Array} The array with all words.
  */
-export default function( text, shouldRemovePunctuation = true ) {
+export default function( text, wordBoundaryRegexString = "\\s", shouldRemovePunctuation = true ) {
 	// Unify whitespaces and non-breaking spaces, remove table of content and strip the tags and multiple spaces.
```

Comment:
Not sure about this, but is it necessary to declare the parameter value there? I couldn't find a case where we override it elsewhere in the code

Reply:
Yes, we override it for example in `buildTopicStems.js`. See also the comment in relevant technical choices about `getWords` for more explanation :) 

And no, `/\s/g` is the same as `new RegExp( "\\s", "g" );`. I guess we could also pass a regex instead of a string that we then turn into a regex. I think the reason I did it this way was because the flag should be global for all regexes, so instead of adding `/g` to every regex we pass, we just add the flag inside the function. But I don't know if that was a good choice 🤷‍♀️